collector/collector.go

```diff
@@ -562,6 +562,17 @@ func parseDateAndTimeWithPattern(metric *config.Metric, pdu *gosnmp.SnmpPDU, met
 	return float64(t.Unix()), nil
 }
 
+func parseNtpTimestamp(pdu *gosnmp.SnmpPDU) (float64, error) {
+	var data []byte = pdu.Value.([]byte)
+
+	// 2208988800 is the Unix Epoch in NTP Seconds (1970 - 1900)
```

Comment:
we should include a hint that the returned data is "Seconds and Fractions since 01.01.1900" as the ntp era starts on jan 1st 1900 whereas unix epoch starts 70y later (so the 2208988800 is basically the offset between those two eras)

Reply:
I have added a comment with additional information, rebased against current main branch and pushed the changes.

Feel free to have a look.

Reply:
🙇